src/content/docs/aws/services/codeartifact.mdx

```diff
@@ -284,44 +282,44 @@ You can configure npm with your CodeArtifact repository without the `awslocal co
   Your repository endpoint is used to point npm to your repository to install or publish packages.
 
     ```bash
-    awslocal codeartifact get-repository-endpoint --domain demo-domain --repository demo-repo --format npm
+    awslocal codeartifact get-repository-endpoint --domain demo-domain --repository demo-repo --format npm --output text
```

Comment:
The default output is configured via AWS profiles, right?

Is it necessary for us to override the output here?

Reply:
By default the output is `json`. By overriding to `text`, you only get the endpoint value.

With the default `json` output:
```
$ awslocal codeartifact get-repository-endpoint --domain demo-domain --repository demo-repo --format npm
{
    "repositoryEndpoint": "http://demo-domain-000000000000.d.codeartifact.eu-central-1.localhost.localstack.cloud:4566/npm/demo-repo/"
}
```

Override to `text` output using `--output text`:
```
$ awslocal codeartifact get-repository-endpoint --domain demo-domain --repository demo-repo --format npm --output text
http://demo-domain-000000000000.d.codeartifact.eu-central-1.localhost.localstack.cloud:4566/npm/demo-repo/
```

If there are multiple fields, you should use `--query repositoryEndpoint` but it is implicit here.